app/package.json

```diff
@@ -61,6 +61,7 @@
     "react-native-splash-screen": "3.3.0",
     "react-native-svg": "12.1.1",
     "react-native-toast-message": "2.0.1",
+    "react-native-unordered-list": "^1.0.4",
```

Comment:
Wondering if we need to install a whole package for this? Doesn't seem like the one referenced is widely used either.